test/gasBenchmarks/executeProposal.js

```diff
@@ -156,7 +156,7 @@ contract('Gas Benchmark - [Execute Proposal]', async (accounts) => {
 
     it('Should execute ERC721 deposit proposal', async () => {
         const depositNonce = 2;
-        const lenMetaData = 0;
+        const lenMetaData = 1;
```

Comment:
Perhaps metadata could be changed to "0x" instead, and length left at 0?

Reply:
yeah